ethereum/gateway-eth-ts/src/contracts/typechain-types/contracts/GatewayToken.ts

```diff
@@ -1152,9 +1145,9 @@ export interface GatewayToken extends BaseContract {
     ): Promise<ContractTransaction>;
 
     "approve(uint256,address,uint256)"(
-      tokenId_: PromiseOrValue<BigNumberish>,
-      to_: PromiseOrValue<string>,
-      value_: PromiseOrValue<BigNumberish>,
+      arg0: PromiseOrValue<BigNumberish>,
```

Comment:
Not sure why the naming is gone here when overwriting, @dankelleher.

Reply:
My overwritten function has no arg names (unused named args are lint errors)